lib/lhm/migrator.rb

```diff
@@ -210,6 +210,7 @@ def destination_create
       original    = %{CREATE TABLE `#{ @origin.name }`}
       replacement = %{CREATE TABLE `#{ @origin.destination_name }`}
       stmt = @origin.ddl.gsub(original, replacement)
+        .gsub(/DEFAULT CHARSET=\w*/, "DEFAULT CHARSET=utf8mb4 ROW_FORMAT=DYNAMIC")
```

Comment:
as far as i know, for us it is always `DEFAULT CHARSET` (i imagine @SingerWang can confirm,) but good call. it is possible that it could just be `CHARSET`. i will add the `CHARSET` case regardless.